docs/USER.md

```diff
@@ -270,6 +271,26 @@ Please note that SPI service didn't perform any specific validity checks
 for the user provided credentials, so it's assumed user provides a correct data.
 
 
+## Overriding default binding lifetime
+In general, bindings (and their dependent secrets) are not supposed to be long-lived. Their TTL is configured globally, 
+and by defaults to 2 hrs. Some tasks may require the credentials to be available for a longer time, so the creator of the binding
+may override the default setting by specifying `spec.lifetime` field of the binding.
+It accepts any standard time periods, like `2h30m`, `90s` etc, except the negative values and values less than
```

Comment:
IMHO, this sentence and the sentence after it are overly complicated.

Reply:
tried to simplify a bit